Loop/Managers/DeviceDataManager.swift

```diff
@@ -117,8 +117,12 @@ final class DeviceDataManager {
         }
 
         loopManager.delegate = self
-        loopManager.carbStore.syncDelegate = remoteDataServicesManager
+
+        loopManager.carbStore.delegate = self
```

Comment:
Might be time to rename DeviceDataManager to LoopManager, have it own the stores, and pass references to LoopDataManager. We can also take a look at that refactoring separately.  Just seems odd for DDM to be reaching into LDM for the stores, and assigning itself as their delegates.

Reply:
I will create a new JIRA story.

Reply:
Done. https://tidepool.atlassian.net/browse/LOOP-947